code/controllers/configuration/configuration.dm

```diff
@@ -25,6 +25,8 @@
 
 	var/list/fail2topic_whitelisted_ips
 
+	var/list/active_donators

```

Comment:
Make sure to protect this list.

Reply:
There's a massive issue that prevents protecting lists.

Regardless, there's redundant measures in place that *in theory* should prevent people from both acquiring and using the donator items of other people, and if someone overrides this list it'll be logged and they'll get a massive paddling.

Unless you want to fix the list security flaw.

Reply:
I have no idea how to fix that flaw.

Reply:
Then this is just going to have to remain unprotected.